test/mainnet-fork/Uniswapv4.t.sol

```diff
@@ -839,6 +839,108 @@ contract MainnetControllerUniswapV4Tests is ForkTestBase {
         assertEq(result.amount1Received, 159.209952e6);
     }
 
+    /**********************************************************************************************/
+    /*** Fuzz Tests                                                                             ***/
+    /**********************************************************************************************/
+
+    /**
+     * @dev Fuzz test for mint + decrease full amounts.
+     */
+    function testFuzz_mintAndDecreaseFullAmounts(
+        int24   tickLower,
+        int24   tickUpper,
+        uint128 liquidity
+    ) external {
+        // Constrain fuzz parameters to valid ranges
+        vm.assume(tickLower < tickUpper);
+        vm.assume(tickLower >= -200 && tickUpper <= 200);
+        vm.assume((tickUpper - tickLower) <= 200); // Max tick spacing
+        vm.assume(liquidity >= 1e6 && liquidity <= 2_000_000e6); // Reasonable liquidity range to avoid rate limit issues
```

Comment:
Use _bound instead of assume to get more running tests

Reply:
done